spyder/plugins/debugger/panels/debuggerpanel.py

```diff
@@ -90,6 +90,7 @@ def paintEvent(self, event):
         super(DebuggerPanel, self).paintEvent(event)
         painter = QPainter(self)
         painter.fillRect(event.rect(), self.editor.sideareas_color)
+        self.paint_cell(painter)
```

Comment:
We had to change the approach with @jsbautista since we realized that calling `super` in some of the panels while adding the `paint_cell` logic in the base class is not possible (the paint operations override between the base class and the subclasses)

Reply:
Ok, no problem then. By living in the `Panel` class, `paint_cell` can be easily called by third-party plugin developers.